beekeeper-api/src/main/java/com/expediagroup/beekeeper/api/controller/BeekeeperController.java

```diff
@@ -106,7 +106,7 @@ public ResponseEntity<Page<HousekeepingPathResponse>> getAllPaths(
           @Spec(path = "cleanupTimestamp", params = "deleted_after", spec = GreaterThan.class),
           @Spec(path = "creationTimestamp", params = "registered_before", spec = LessThan.class),
           @Spec(path = "creationTimestamp", params = "registered_after", spec = GreaterThan.class) }) Specification<HousekeepingPath> spec,
-      Pageable pageable) {
+      @ParameterObject Pageable pageable) {
```

Comment:
Added @ParameterObject to the Pageable parameter, this allows Swagger to automatically generate and display pagination parameters (page, size, sort) for /unreferenced-paths. This will make it consistent with /metadata and reduces manual parameter definitions.

Reply:
isn't this for requests params e.g. `url/bla?myRequestParam=foo`
I thought the Pageable is part of the request body in this API?

Reply:
The `Pageable` parameter is populated from url query params. The `GET` request won't have a request body but the json format in swagger is misleading . . 

The `@ParameterObject` will ensure swagger generates better documentation for the query params, we now have separate fields for `Page`, `Size` & `Sort`. It also resolves the `Sort` filter issue by making it clearer and ensuring it's optional by default, which wasn’t the case in the JSON format.                                                                           

We already have this in the `/metadata` endpoint so not sure why it wasn't done for this one, you can see how the layout differs between endpoints in swagger.

Reply:
cool thanks for explaning!